src/Tools/dotnet-trace/Extensions.cs

```diff
@@ -12,7 +12,41 @@ namespace Microsoft.Diagnostics.Tools.Trace
 {
     internal static class Extensions
     {
+        public static string CLREventProviderName = "Microsoft-Windows-DotNETRuntime";
+
         private static EventLevel defaultEventLevel = EventLevel.Verbose;
+        // Keep this in sync with runtime repo's clretwall.man
```

Comment:
This thought isn't in the scope of this PR, but you and I have made comments like this pretty routinely in our code.  I think we may want to discuss a way to automate the generation of these _types_ of constructs.  The .man file is very highly structured so I imagine we could script around it easily.

Reply:
Yeah... It's also in PerfView. Maybe we can do something like the dependency update bot does for arcade? The main challenge I see is that there's bits of logic involved in every repo. The .man file is structured, but the .cs files that consume these from PerfView or diagnostics repo may not be. I'm always up for discussion on how to keep these consistent though!